### DISCUSSION TITLE
Consulration Request about ${SERVER_PUB_NIC}

### DISCUSSION THREAD

**Discussion by Sinsstranger** [complexorganizations/wireguard-manager#304] [General]
What rule should I add to wireguard-manager.sh script for ${SERVER_PUB_NIC} on my server will not answer to ping? Unfortunatly I do not familiar with nftables...(

**Comment by ghost** [comment]
U can ignore the nftables update, iptables will work fine.

also here is the answer.
```
SERVER_PUB_NIC="$(ip -4 route ls | grep default | grep -Po '(?<=dev )(\S+)' | head -1)"
```

**Reply by Sinsstranger** [reply]
I mean that i do not know what to do with vpn server answers to ping while wireduard is up. And I'm interested in how block it via nftables according with wireguard-manager script logic. Simplier: I need than wireguard is working, my eth0 interface don't answer to ping from internet. Sorry to interupt)

**Reply by ghost** [reply]
Join this signal chat please, I need a bit more info.

https://signal.group/#CjQKIIU_YFbm27-G08aYUzncE9AQZQ4RIJF_xvTfGy-721GdEhAsu9_5KuzNhgxNE5yDiKfH

**Reply by Sinsstranger** [reply]
ok, I will try it later...(I am at work...)

**Reply by ghost** [reply]
> 

Please join as soon as possible, i want to get this issue solved.
